### ISSUE TITLE
Specify relations between registers and contained items

### ISSUE THREAD

**Issue by strogonoff** [relaton/relaton-iana#10]
The idea is to do this:

- [ ] Create “includes” relations from a “root” bibitem (e.g., docid `bundle`) to each nested bibitem (e.g., docid `bundle/admin-record-types` and all other `bundle/*` items)
- [ ] Create one “includedIn” relation from each nested bibitem (e.g., `bundle/admin-record-types`) to its container (e.g., `bundle`)

**Comment by ronaldtse** [comment]
I agree with this structure. Basically all leaf nodes are bibitems, every non-leaf node is a registry (or a dataset).

**Comment by strogonoff** [comment]
To avoid making category errors, all of them are bibitems—but the types are different: for non-leaf registry nodes it’s “dataset”, while leaf nodes can be something else (“standard”, etc.).

Also, as I understand, in relaton-data-iana we don’t have bibitems for the actual leaf nodes (registry items) yet. All we have for now is registries: sub-registries (e.g. foo) and sub-sub-registries (foo/bar). A registry item would be foo/bar/baz, but we don’t have those as separate items.

**Comment by ronaldtse** [comment]
@strogonoff ah sorry for the poor terminology. Lack of sleep!

We could have bibitems for all of the nodes, but as discussed with @rjsparks, we only need to have bibitems for registries and sub-registries.

**Comment by strogonoff** [comment]
Understood. @andrew2net do you think you could implement this change? According to our chat with Nick, it probably shouldn’t break anything.

**Comment by andrew2net** [comment]
@strogonoff could you describe the changes with a diagram or data structure (XML/YAML) examples?

**Comment by strogonoff** [comment]
@andrew2net, I corrected ticket description and title after the discussion.
